<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChainConfigTabWidget</class>
 <widget class="QWidget" name="ChainConfigTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget,
QDialog
{
	background-color: rgb(255, 255, 255);
	font-size: 9pt;
}


#wgtChainConfigs, #wgtMultisensors
{
	min-width: 360px;
	max-width: 360px;
	min-height: 80px;
}


QPushButton
{
	background-color: rgb(230, 230, 230);
	border: none;
	border-radius: 6px;
	min-width: 140px;	
	min-height: 24px;
}

QPushButton:hover
{
	border-top: 1px solid black;
	border-left: 1px solid black;
	border-right: 2px outset black;
	border-bottom: 2px outset black;
}

QPushButton:pressed
{
	border: 2px inset black;
}


QComboBox
{
	background-color: rgb(230, 230, 230);
	border-radius: 6px;
	border: 1px solid gray;
	min-width: 120px;
	min-height: 18px;
}


QSpinBox
{
	background-color: rgb(230, 230, 230);
	border-radius: 6px;
	border: 1px solid gray;
	min-height: 20px;
	min-width: 60px;
}


QLineEdit
{
	background-color: rgb(230, 230, 230);
	border-radius: 6px;
	border: 1px solid gray;
}


#btnSaveAnalogChainSettings,
#btnSaveDigitalChainSettings,
#btnWriteMsSettings
{
	min-width: 170px;
	padding: 0px 12px;
}


QGroupBox::enabled
{
	color: black;
}


!QGroupBox::enabled
{
	color: gray;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="IssoEditListPanel" name="wgtChainConfigs" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbSettings">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Настройки</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>30</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Тип шлейфа:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbtnSmoke">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Дым</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbtnTemp">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Температура</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbtnAlarmButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Тревожная кнопка</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbtnMoveDetector">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Детектор движения</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbtnMulticriterial">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Мультикритериальный</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stwgtChainSettings">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="pageAnalogChainSettings">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QWidget" name="wgtNormal_2" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>330</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="0,0">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_92">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="text">
                <string>Количество датчиков</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="sbSensorCount"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="IssoAnalogSensorSetupPanel" name="wgtAnalogChainSetup" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>330</width>
              <height>200</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QPushButton" name="btnSaveAnalogChainSettings">
              <property name="text">
               <string>Сохранить настройки</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageDigitalChainSettings">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>30</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="IssoEditListPanel" name="wgtMultisensorConfigs" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>7</number>
              </property>
              <item>
               <widget class="QFrame" name="frmMsType">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,2">
                 <property name="spacing">
                  <number>10</number>
                 </property>
                 <property name="leftMargin">
                  <number>10</number>
                 </property>
                 <property name="topMargin">
                  <number>10</number>
                 </property>
                 <property name="rightMargin">
                  <number>4</number>
                 </property>
                 <property name="bottomMargin">
                  <number>10</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Тип ИПМ</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="cbMsType">
                   <property name="minimumSize">
                    <size>
                     <width>122</width>
                     <height>20</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>Внутренние датчики</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <property name="leftMargin">
                  <number>20</number>
                 </property>
                 <item>
                  <widget class="QWidget" name="wgtMultisensorSettings" native="true">
                   <layout class="QVBoxLayout" name="verticalLayout_3">
                    <property name="spacing">
                     <number>7</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>6</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>6</number>
                    </property>
                    <item>
                     <widget class="IssoDigitalSensorSetupPanel" name="wgtSmokeOptical" native="true"/>
                    </item>
                    <item>
                     <widget class="QGroupBox" name="gbSmokeOpticalExt">
                      <property name="enabled">
                       <bool>true</bool>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="title">
                       <string>Дым (о) - расширенные настройки</string>
                      </property>
                      <property name="checkable">
                       <bool>false</bool>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                      <layout class="QHBoxLayout" name="horizontalLayout_7">
                       <property name="spacing">
                        <number>10</number>
                       </property>
                       <property name="leftMargin">
                        <number>20</number>
                       </property>
                       <item>
                        <widget class="QLabel" name="label_2">
                         <property name="text">
                          <string>Чувствительность</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QSpinBox" name="sbSmokeOptSens">
                         <property name="minimumSize">
                          <size>
                           <width>62</width>
                           <height>22</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>100</width>
                           <height>16777215</height>
                          </size>
                         </property>
                         <property name="maximum">
                          <number>65535</number>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <spacer name="horizontalSpacer_3">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeType">
                          <enum>QSizePolicy::Fixed</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>20</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_4">
                         <property name="text">
                          <string>Яркость</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QSpinBox" name="sbSmokeOptBright">
                         <property name="minimumSize">
                          <size>
                           <width>62</width>
                           <height>22</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>100</width>
                           <height>16777215</height>
                          </size>
                         </property>
                         <property name="maximum">
                          <number>65535</number>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <spacer name="horizontalSpacer_6">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeType">
                          <enum>QSizePolicy::Expanding</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>20</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item>
                     <widget class="IssoDigitalSensorSetupPanel" name="wgtTempAnalog" native="true"/>
                    </item>
                    <item>
                     <widget class="IssoDigitalSensorSetupPanel" name="wgtTempDigital" native="true"/>
                    </item>
                    <item>
                     <widget class="IssoDigitalSensorSetupPanel" name="wgtSmokeElectrochemical" native="true"/>
                    </item>
                    <item>
                     <widget class="IssoDigitalSensorSetupPanel" name="wgtFlameMinValue" native="true"/>
                    </item>
                    <item>
                     <widget class="IssoDigitalSensorSetupPanel" name="wgtFlameStdDev" native="true"/>
                    </item>
                    <item>
                     <widget class="IssoDigitalSensorSetupPanel" name="wgtCO" native="true"/>
                    </item>
                    <item>
                     <widget class="IssoDigitalSensorSetupPanel" name="wgtFlyMatter" native="true"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="gbPeripherals">
                <property name="title">
                 <string>Периферия</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <property name="spacing">
                  <number>10</number>
                 </property>
                 <property name="leftMargin">
                  <number>20</number>
                 </property>
                 <property name="topMargin">
                  <number>9</number>
                 </property>
                 <property name="rightMargin">
                  <number>9</number>
                 </property>
                 <property name="bottomMargin">
                  <number>9</number>
                 </property>
                 <item>
                  <widget class="QCheckBox" name="chkCameraEnabled">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>Камера</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="cbCamera">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="gbLocationDesc">
                <property name="title">
                 <string>Местоположение</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <property name="leftMargin">
                  <number>9</number>
                 </property>
                 <property name="topMargin">
                  <number>9</number>
                 </property>
                 <property name="rightMargin">
                  <number>9</number>
                 </property>
                 <property name="bottomMargin">
                  <number>9</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="leLocationDesc"/>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QPushButton" name="btnSaveDigitalChainSettings">
              <property name="text">
               <string>Сохранить настройки</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IssoEditListPanel</class>
   <extends>QWidget</extends>
   <header>issoeditlistpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IssoAnalogSensorSetupPanel</class>
   <extends>QWidget</extends>
   <header>issoanalogsensorsetuppanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IssoDigitalSensorSetupPanel</class>
   <extends>QWidget</extends>
   <header>issodigitalsensorsetuppanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>chkCameraEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbCamera</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
